docs/source/index.mdx

```diff
@@ -1,76 +1,38 @@
-
-<br>
-
-## Inference API integration into the Hugging Face Hub
-
-In order to get a functional Inference API on the Hub for your models (and thus, cool working widgets!) check out this [doc](https://github.com/huggingface/huggingface_hub/tree/main/api-inference-community). There is a docker image for each library. Within the image, you can find the implementation for supported pipelines for the given library.
-
-<br>
-
-
-## Widgets
-
-All our widgets are open-sourced. Feel free to propose and implement widgets. You can try all of them out [here](https://huggingface-widgets.netlify.app/).
-
-
-<br>
-
-## Code Snippets
-
-We'll implement a few tweaks to improve the UX for your models on the website – let's use [Asteroid](https://github.com/asteroid-team/asteroid) as an example.
-
-Model authors add an `asteroid` tag to their model card and they get the advantages of model versioning built-in
-
-![asteroid-model](docs/assets/asteroid_repo.png)
-
-We add a custom "Use in Asteroid" button. When clicked, you get a library-specific code sample that you'll be able to specify. 🔥
-
-![asteroid-code-sample](docs/assets/asteroid_snippet.png)
-
-
-<br>
-
-## Feedback (feature requests, bugs, etc.) is super welcome 💙💚💛💜♥️🧡
+# Hub client library
+
+The `huggingface_hub` library allows you to interact with the [Hugging Face
+Hub](https://hf.co), a machine learning platform for creators and collaborators.
+Discover pre-trained models and datasets for your projects or play with the hundreds of
+machine learning apps hosted on the Hub. You can also create and share your own models
+and datasets with the community. The `huggingface_hub` library provides a simple way to
+do all these things with Python.
+
+Read the [quick start guide](quick-start) to get up and running with the
+`huggingface_hub` library. You will learn how to download files from the Hub, create a
+repository, and upload files to the Hub.
+
+Keep reading to learn even more about how to:
+
+- [Delete and clone a repository, and create and update a branch](how-to-manage).
+- [Download an entire repository and use regex matching to filter and download specific
+  files](how-to-downstream).
+- [Upload your files with a context manager or use a helper to push files to a remote
+  repository](how-to-upstream).
+- [Search thousands of models and datasets on the Hub with specific filters and
+  parameters to only return the best results](searching-the-hub).
+- [Access the Inference API for accelerated inference](how-to-inference).
+
+## Contribute
+
+All contributions to the `huggingface_hub` are welcomed and equally valued! 🤗 Besides
+adding or fixing existing issues in the code, you can also help improve the
+documentation by making sure it is accurate and up-to-date, help answer questions on
+issues, and request new features you think will improve the library. Take a look at the
+[contribution
```

Comment:
It'd be nice to have both this contributing guide and our CoC rendered in the docs page. Not in this PR, but something to consider as a future PR.